package.json

```diff
@@ -236,6 +236,10 @@
                 "description": "Azure SQL Copilot agent",
                 "isSticky": true,
                 "commands": [
+                    {
+                        "name": "help",
+                        "description": "Provides a list of available commands and their descriptions."
```

Comment:
Nit: "Provide" (no 's') to match the rest of the command descriptions.


... technically, `getConnectionDetails` should be `Show a...` instead of `Shows a...`, too

Reply:
updated both